pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -207,6 +212,46 @@ func eventFromChannel(c chan string, testName string, wantEvents []string) error
 	return nil
 }
 
+// getTaskRunCreations will look through a set of actions to find all task run creation actions and return the set of
+// them. It will fatal the test if none are found.
+func getTaskRunCreations(t *testing.T, actions []ktesting.Action) []*v1beta1.TaskRun {
+	outputs := []*v1beta1.TaskRun{}
```

Comment:
We should put a `t.Helper()` here so that the error report points to the caller instead of the `t.Fatal` in this function